web/assets/css/transmission-app.scss

```diff
@@ -1364,19 +1365,15 @@ $video-image: '../img/film.svg';
     0 9px 28px 8px #0000000d;
   color: var(--color-fg-on-popup);
   min-width: 220px;
-  overflow: auto;
+  overflow: scroll;
   -webkit-overflow-scrolling: touch;
   padding: 10px;
```

Comment:
This changes padding on the menu item in a non-even way. Any particular reason?

Reply:
So there's no gap left in the right where the browser's own space bar will take over. Whatever behind that small gap aren't meant to be interacted with!

Reply:
Forgot that right: 0px will close the gap and this will make some clearance for browser's own scroll bar to take over.

Reply:
So, my OS’s scrollbars don’t cause an issue. (And I think it’s because 10px, and the current minimum width, are totally fine as-is.) Would you mind showing yours?

<img width="165" alt="Screenshot 2023-07-25 at 10 41 46 AM" src="https://github.com/transmission/transmission/assets/435896/c9fa8cda-f0ab-4728-bd9e-f8fb13a2bf8f">
<img width="258" alt="Screenshot 2023-07-25 at 10 41 39 AM" src="https://github.com/transmission/transmission/assets/435896/2e899db3-714d-41dd-b03f-943d17f9cb5f">

Reply:
I meant the primary scroll bar

Reply:
I also don’t see any issues there, even when macOS goes to “huge” scrollbars, with a double-scroll scenario:

<img width="121" alt="Screenshot 2023-07-25 at 10 49 30 AM" src="https://github.com/transmission/transmission/assets/435896/969145f1-e438-49d7-918e-88cfc00f8b9f">

Reply:
Having tested again as I realized my proposed change might have primary and secondary scrolls overlapped to each other which unfortunately it did, my point was to not let the gap behind having an interaction, you wouldn't want to fat finger to start a torrent instead when trying to drag the scroll bar. ;)